mathics/builtin/box/graphics.py

```diff
@@ -114,7 +115,7 @@ def init(self, graphics, style, item):
                 else:
                     self.arc = (start_angle, end_angle)
 
-            item = Expression(item.get_head_name(), *item.elements[:2])
+            item = Expression(Symbol(item.get_head_name()), *item.elements[:2])
```

Comment:
BTW - Symbol(element.get_head_name()) comes up often enough that we probably should arrange all simple or non-compound elements to have their symbols stored. 

I am sure there is a way to do this automatically for things like those things derived off of Bultin, but I don't know off hand. 

One of the many things for down the line.